Misc/NEWS.d/next/Library/2024-09-01-20-20-49.gh-issue-78981.nNRvSX.rst

```diff
@@ -0,0 +1,2 @@
+Fixed a bug where ``email.contentmanager.set_content`` raised error when
```

Comment:
The method should be modified with `:meth:`.
```
:meth:`email.contentmanager.set_content`
```

Reply:
Updated